client/webpack/webpackConfig.dev.js

```diff
@@ -102,6 +103,8 @@ module.exports = {
 		new ManifestPlugin({
 			publicPath: '/dist/',
 		}),
+		// Allow shared utils to import the sentry/node package by replacing it in the webpack build
+		new webpack.NormalModuleReplacementPlugin(/@sentry\/node/, '@sentry/react'),
```

Comment:
Without this, the webpack build fails when it tries to import the node sentry sdk rather than the react one in the new assert implementation. Since they have the same API, this works fine even if it feels like a hack!